website/views/static/bug.jade

```diff
@@ -0,0 +1,21 @@
+extends ./layout
+
+block vars
+  - var layoutEnv = env
+  - var menuItem = 'bug'
+
+block title
+  title Habitica &VerticalLine;&nbsp;
+    =env.t('communityBug')
+
+block content
+  .row
+    .col-md-12
+      .page-header
+        h1=env.t('communityBug')
+      p
+        =env.t('tryFirst')
+        button(onclick='localStorage.clear()') Clear local storage
```

Comment:
This should say "Clear data" and be a translatable string.

Please also include clearing cookies. Although I suppose if that's complex, we could remove mention of cookies from the text I've just suggested and go live with local storage only, and then add cookies later. Local storage is the main thing that I want to have cleared. Cookies tend to be easier to clear with built-in browser features.